src/routes/search/FacetCard.svelte

```
@@ -1,3 +1,4 @@
+import { t } from '$lib/i18n';
```

ESLint

[error] 't' is not defined.

(no-undef)